machine/views/res_partner_views.xml

```diff
@@ -1,11 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <odoo>
     <record id="machine_instance_partner_action" model="ir.actions.act_window">
-        <field name="name">Machine Instances</field>
+        <field name="name">Machines</field>
         <field name="res_model">machine.instance</field>
         <field name="domain">[('partner_id', '=', active_id)]</field>
```

Comment:
Maybe this should filter per commercial partner?

Reply:
Maybe, but then this should be done differently. We only show `Machines` smart button if there is at least one machine created, because most partners won't have machine for them, so its better to not show that button for em. So all we need is to just filter `partner_id` from within machine.